src/web/components/global-nav/side-nav/link/link.scss

```diff
@@ -0,0 +1,18 @@
+@import '~basics/colors/colors.scss';
+
+.hig__global-nav__sidenav__links {
+  font-size: 12px;
+  line-height: 26px;
+
+  .hig__global-nav__sidenav__links__link {
```

Comment:
I don't see a reason to nest this selector under `.hig__global-nav__sidenav__links`.